ui-admin/src/study/surveys/editor/SurveyPreview.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react'
+import { SurveyModel } from 'survey-core'
+import { Survey as SurveyJSComponent } from 'survey-react-ui'
+
+import { extractSurveyContent, JuniperSurvey, Survey } from '@juniper/ui-core'
+
+type SurveyPreviewProps = {
+  survey: JuniperSurvey
+}
+
+export const SurveyPreview = (props: SurveyPreviewProps) => {
+  const { survey } = props
+
+  const [surveyModel] = useState(() => {
+    const fakeForm = { content: JSON.stringify(survey) } as Survey
+
+    const model = new SurveyModel(extractSurveyContent(fakeForm))
+    model.focusFirstQuestionAutomatic = false
+    model.showTitle = false
```

Comment:
showTitle = false and widthMode static are also set on the participant side -- do we ever not want these?  And if we don't, should we pull these into a "configureDefaultSurveyDisplay" utility method somewhere?